cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CurrentLimitsMapping.java

```diff
@@ -32,10 +32,12 @@ public CurrentLimitsAdder getCurrentLimitsAdder(String id, Supplier<CurrentLimit
     void addAll() {
         for (Map.Entry<String, CurrentLimitsAdder> entry : adders.entrySet()) {
             if (Double.isNaN(entry.getValue().getPermanentLimit())) {
-                context.ignored("Operational Limit Set of " + entry.getKey(), "An operational limit set must at least contain one value for permanent limit.");
-            } else {
-                entry.getValue().add();
+                context.fixed("Operational Limit Set of " + entry.getKey(),
```

Comment:
The PATL seems to be mandatory in CGMES for all transformers and at both ends for a AC line. So I am not sure that it is safe just to replace them by infinite values. What do you think ?

Reply:
In theory, this is true, but it seems that in French networks, current TATL are assigned without PATL...

Reply:
We can take the highest value of TATL as PATL though? It may be more correct that putting an infinite PATL

Reply:
No, I was more thinking about a warning.